PowerEditor/installer/APIs/actionscript.xml

```diff
@@ -25,320 +22,211 @@
 		<KeyWord name="bullet" />
 		<KeyWord name="Button" />
 		<KeyWord name="call" />
-		<KeyWord name="call" />
 		<KeyWord name="callee" />
 		<KeyWord name="caller" />
 		<KeyWord name="Camera" />
 		<KeyWord name="capabilities" />
 		<KeyWord name="Capabilities" />
 		<KeyWord name="CAPSLOCK" />
-		<KeyWord name="CAPSLOCK" />
 		<KeyWord name="case" />
 		<KeyWord name="ceil" />
-		<KeyWord name="ceil" />
 		<KeyWord name="charAt" />
-		<KeyWord name="charAt" />
-		<KeyWord name="charCodeAt" />
 		<KeyWord name="charCodeAt" />
 		<KeyWord name="childNodes" />
-		<KeyWord name="childNodes" />
-		<KeyWord name="chr" />
 		<KeyWord name="chr" />
 		<KeyWord name="clear" />
 		<KeyWord name="clearInterval" />
 		<KeyWord name="cloneNode" />
-		<KeyWord name="cloneNode" />
 		<KeyWord name="close" />
-		<KeyWord name="close" />
-		<KeyWord name="Color" />
 		<KeyWord name="color" />
 		<KeyWord name="Color" />
 		<KeyWord name="concat" />
-		<KeyWord name="concat" />
 		<KeyWord name="condenseWhite" />
 		<KeyWord name="connect" />
-		<KeyWord name="connect" />
-		<KeyWord name="constructor" />
 		<KeyWord name="constructor" />
 		<KeyWord name="contentType" />
 		<KeyWord name="continue" />
-		<KeyWord name="continue" />
-		<KeyWord name="CONTROL" />
 		<KeyWord name="CONTROL" />
 		<KeyWord name="Cookie" />
 		<KeyWord name="cos" />
-		<KeyWord name="cos" />
-		<KeyWord name="createElement" />
 		<KeyWord name="createElement" />
 		<KeyWord name="createEmptyMovieClip" />
 		<KeyWord name="createTextField" />
 		<KeyWord name="createTextNode" />
-		<KeyWord name="createTextNode" />
 		<KeyWord name="curveTo" />
 		<KeyWord name="CustomActions" />
 		<KeyWord name="CustomActions.get" />
 		<KeyWord name="CustomActions.install" />
 		<KeyWord name="CustomActions.list" />
 		<KeyWord name="CustomActions.uninstall" />
 		<KeyWord name="data" />
-		<KeyWord name="data" />
-		<KeyWord name="Date" />
 		<KeyWord name="Date" />
 		<KeyWord name="Date.UTC" />
 		<KeyWord name="decode" />
 		<KeyWord name="default" />
 		<KeyWord name="delete" />
-		<KeyWord name="delete" />
 		<KeyWord name="DELETEKEY" />
-		<KeyWord name="DELETEKEY" />
-		<KeyWord name="do" />
 		<KeyWord name="do" />
 		<KeyWord name="docTypeDecl" />
-		<KeyWord name="docTypeDecl" />
 		<KeyWord name="domain" />
 		<KeyWord name="DOWN" />
-		<KeyWord name="DOWN" />
-		<KeyWord name="dragOut" />
 		<KeyWord name="dragOut" />
 		<KeyWord name="dragOver" />
-		<KeyWord name="dragOver" />
-		<KeyWord name="duplicateMovieClip" />
 		<KeyWord name="duplicateMovieClip" />
 		<KeyWord name="duration" />
 		<KeyWord name="else" />
-		<KeyWord name="else" />
 		<KeyWord name="embedFonts" />
 		<KeyWord name="enabled" />
 		<KeyWord name="END" />
-		<KeyWord name="END" />
 		<KeyWord name="endFill" />
 		<KeyWord name="ENTER" />
-		<KeyWord name="ENTER" />
-		<KeyWord name="enterFrame" />
 		<KeyWord name="enterFrame" />
 		<KeyWord name="eq" />
 		<KeyWord name="escape" />

```

Comment:
Line 166 and 169 look identical.

Reply:
Again just ran a regex to find duplicates next to each other so it doesn't work if the list isn't properly sorted.

Reply:
I feel this also should be removed.

![image](https://user-images.githubusercontent.com/14791461/33807618-a0b9b83a-ddff-11e7-9365-ba05cf4db6b4.png)